k8s/overlays/azure/dev/env.properties

```
@@ -80,6 +80,15 @@ GRIDCAPA_MAX_POLLED_FILES_CSE_EXPORT_IDCC=10
 GRIDCAPA_MINIO_BUCKET_CSE_EXPORT_IDCC=gridcapa
 GRIDCAPA_MINIO_BASE_DIRECTORY_CSE_EXPORT_IDCC=/CSE/EXPORT/IDCC
 
+GRIDCAPA_SOURCE_HOST_SWE_BTCC=ftp-server
+GRIDCAPA_SOURCE_PORT_SWE_BTCC=21
+GRIDCAPA_SOURCE_BASE_DIRECTORY_SWE_BTCC=/data/swe/btcc
+GRIDCAPA_SINK_DIRECTORY_SWE_BTCC=data/gridcapa/swe/btcc/outputs
```

I believe sink directory should rather be : 
```suggestion
GRIDCAPA_SINK_DIRECTORY_SWE_BTCC=/dataswe/btcc/outputs
```